### DISCUSSION TITLE
Methods to skip .dex files in assets dir

### DISCUSSION THREAD

**Discussion by lihe07** [iBotPeaches/Apktool#2914] [Q&A] [answered]
I have an apk with following structure:
- /
- classes.dex
- classes2.dex
- classes3.dex
- /assets
  - 39285EFA.dex

The `39285EFA.dex` in assets seems not like an correct dex file, and when I decompiling the apk, I got an error:
```
I: Baksmaling assets/39285EFA.dex...
Exception in thread "main" org.jf.dexlib2.dexbacked.DexBackedDexFile$NotADexFile: Not a valid dex magic value: cf 77 4c c7 9b 21 01 cd
```

**Comment by harshitshah4** [comment, marked as answer]
Have you tried sending `--only-main-classes` flag
